gnovm/stdlibs/std/emit_event.go

```diff
@@ -0,0 +1,77 @@
+package std
+
+// ref: https://github.com/gnolang/gno/pull/853
```

Comment:
```suggestion
// ref: https://github.com/gnolang/gno/pull/575
```

Let's refer the initial issue and make sure that it's up-to-date.